public/services/twitter.service.js

```diff
@@ -36,6 +38,18 @@ angular.module('textAnalysis')
                             });
                     }
                     return deferred.promise;
+                },
+                getSingleTweet: function(tweetId) {
+                    var deferred = $q.defer();
+                    $http.get('/api/results/' + $routeParams.id)
+                        .success(function(data) {
+                            deferred.resolve(data);
+                            twitterObj.resultsTweet = data;
+                        }).error(function(e) {
+                            console.log('Error: ', e);
```

Comment:
And this one.  Use $log instead if you need to output some debugging info.  Inject `$log` into your controller/service and replace console with `$log`, so an example would be `$log.log('Error: %s', e)`